### PR TITLE
Update user stories and use cases in DeveloperGuide.md

### LINKED ISSUE #18: Update the users stories and use cases

### PR DESCRIPTION
**Pull request by nguyendqminh**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -86,9 +86,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. TrackIt@NUS shows an error message.
 
       Use case resumes at step 2.
+      
 
 *{More to be added}*
```
Comment (simonteozw) [inline docs/DeveloperGuide.md]: Should we remove this `*{More to be added}*` line?

Comment (nguyendqminh) [reply docs/DeveloperGuide.md]: I am not sure; we will update this section constantly, but the line might be unnecessary. I'll remove it for now.

Comment (nguyendqminh) [reply docs/DeveloperGuide.md]: Update: fixed.

**Review by UncleGrandpa925 — APPROVED** [review]
Okay looks good to me

**Pull request merged**